CRM/Membershipextrasimporterapi/CSVRowImporter.php

```diff
@@ -17,6 +17,9 @@ public function import() {
 
     $membershipImporter = new CRM_Membershipextrasimporterapi_EntityImporter_Membership($this->rowData, $this->contactId, $recurContributionId);
     $membershipId = $membershipImporter->import();
+
+    $contributionImporter = new CRM_Membershipextrasimporterapi_EntityImporter_Contribution($this->rowData, $this->contactId, $recurContributionId);
+    $contributionId = $contributionImporter->import();
```

Comment:
$membershipId and $contributionId are never used. I'm guessing you'll need them later to create the ine item?

Reply:
yup